cypher/cypher-docs/src/docs/dev/ql/administration/security/security-and-indexes-intro.asciidoc

```diff
@@ -102,24 +119,33 @@ Now we run the same queries again:
 MATCH (n:User) WHERE n.name CONTAINS 'ndy' RETURN n.name;
 ----
 
-This query will operate exactly as before, returning the same single result, because nothing in this query relates to the denylisted property.
+This query will operate exactly as before, returning the same two results, because nothing in this query relates to the blacklisted property.
 
-But consider the full-text index query:
+However, for the query targeting the composite index things have changed.
+
+[source, cypher]
+----
+MATCH (n:User) WHERE n.name CONTAINS 'ndy' AND n.surname IS NOT NULL RETURN n.name;
+----
+
+Since the `surname` property is blacklisted it will appear to always be `null` and the composite index empty. Therefore, query returns no result.
```

Comment:
```suggestion
Since the `surname` property is denylisted, it will appear to always be `null` and the composite index empty. Therefore, the query returns no result.
```